doc/man3/CMS_add0_cert.pod

```diff
@@ -19,25 +20,30 @@ CMS_add0_cert, CMS_add1_cert, CMS_get1_certs, CMS_add0_crl, CMS_add1_crl, CMS_ge
 
 =head1 DESCRIPTION
 
-CMS_add0_cert() and CMS_add1_cert() add certificate B<cert> to B<cms>.
-must be of type signed data or enveloped data.
+CMS_add0_cert() and CMS_add1_cert() add certificate I<cert> to I<cms>.
+I<cms> must be of type signed data or (authentiated) enveloped data.
+For signed data, such a certificate can be used to fill in the signer
+certificate and may also be helpful for chain building in certificate valiation.
 
-CMS_get1_certs() returns all certificates in B<cms>.
+CMS_get1_certs() returns all certificates in I<cms>.
 
-CMS_add0_crl() and CMS_add1_crl() add CRL B<crl> to B<cms>. CMS_get1_crls()
-returns any CRLs in B<cms>.
+CMS_add0_crl() and CMS_add1_crl() add CRL I<crl> to I<cms>.
+I<cms> must be of type signed data or (authentiated) enveloped data.
+For signed data, such a CRL may be used in certificate valiation.
+
+CMS_get1_crls() returns all CRLs in I<cms>.
 
 =head1 NOTES
 
-The CMS_ContentInfo structure B<cms> must be of type signed data or enveloped
-data or an error will be returned.
+The CMS_ContentInfo structure I<cms> must be of type signed data or enveloped
+data or authenticated enveloped data or an error will be returned.
 
-For signed data certificates and CRLs are added to the B<certificates> and
-B<crls> fields of SignedData structure. For enveloped data they are added to
-B<OriginatorInfo>.
+For signed data, certificates and CRLs are added to the B<certificates> and
```

Comment:
`I<certificates>`

Reply:
I thought that field names, like type names, are also indicated by `B<>`,
but now I've learned that for field names this not the case. Fixed.